packages/quantic/force-app/main/default/lwc/quanticRecentResultsList/e2e/fixture.ts

```diff
@@ -0,0 +1,38 @@
+import {RecentResultListObject} from './pageObject';
+import {quanticBase} from '../../../../../../playwright/fixtures/baseFixture';
+import {SearchObject} from '../../../../../../playwright/page-object/searchObject';
+import {searchRequestRegex} from '../../../../../../playwright/utils/requests';
+
+const pageUrl = 's/quantic-recent-results-list';
+
+interface RecentQueriesListOptions {
```

Comment:
Please do a `ctrl + f` of the word `queries` in all the files changed in this PR to make sure none was forgotten.
```suggestion
interface RecentResultsListOptions {
```